### ISSUE TITLE
[OSX] Browser window empty after restore from dock

### ISSUE THREAD

**Issue by gmcinalli** [sieren/QSyncthingTray#220]
Syncthing browser window appear empty when minimized to dock and then restored.
I also noticed that if then I modify the window size the Syncthing dashboard re-appears. 

Before minimizing:
![image](https://user-images.githubusercontent.com/9024983/28616425-fda18a10-71fc-11e7-99a7-18928a2b2074.png)

After minimizing:
![image](https://user-images.githubusercontent.com/9024983/28616601-9b644f44-71fd-11e7-988c-14b33da488fd.png)

**Comment by sieren** [comment]
Are you still being able to reproduce this? Which OSX version are you on?

**Comment by gmcinalli** [comment]
Yes. I'm now on High Sierra 10.13.2, when I opened the ticket I was on Sierra.

**Comment by sieren** [comment]
I think this might be a Qt bug. I can reproduce it with the current release, but it seems gone when compiling with Qt 5.9. So I expect this to be fixed with the next release which I'm gonna put out in the next few days.

**Comment by gmcinalli** [comment]
Ok, perfect.
